18-minimal/Dockerfile.c8s

```diff
@@ -1,4 +1,4 @@
-FROM ubi8/ubi-minimal:latest
+FROM quay.io/sclorg/s2i-core-c8s
```

Comment:
Nodejs minimal images have dependencies to ubi8/ubi-minimal image. Why this is replaced? I don't follow.
As far as I know ubi8/ubi-minimal is publically available ike the other UBI images, right?

Reply:
Yes, both is available, but for `Dockerfile.c8s` we want to install CentOS Stream 8 packages, therefore we need a base image with C8S repos. And we should not mix ubi (RHEL) with CentOS Stream builds.